fe/fe-core/src/main/java/org/apache/doris/planner/SingleNodePlanner.java

```diff
@@ -1713,6 +1713,7 @@ private PlanNode createScanNode(Analyzer analyzer, TableRef tblRef, SelectStmt s
                 break;
         }
         if (scanNode instanceof OlapScanNode || scanNode instanceof EsScanNode || scanNode instanceof HiveScanNode) {
+            PredicatePushDown.visitScanNode(scanNode, tblRef.getJoinOp(), analyzer);
```

Comment:
You should fix the 'InferFiltersRule' rather than add visit in here ~
The logic of `predicate propagate` in the  'InferFiltersRule'.
The 'Predicate Push Down' is the abandoned logic ~

Reply:
yes, u r right, but we need a quick fix for predicate propagate before fix the 'InferFiltersRule' since there is 10 times slower in specific scene. i will fix 'InferFiltersRule' later.

Reply:
If it's a temporary fix, I don't think it should be merged into the trunk.
And we can give users a temporary fix release to solve their problem.

Reply:
that's ok for this case. but I strongly recommend not removing any performance-related features until alternatives are fully developed